dense_tiles/dense_tiles.cpp

```diff
@@ -139,6 +165,10 @@ int main(int argc, char* argv[]) {
     for (unsigned int i = 0; i < grid.size(); ++i) {
         if (grid[i]) {
             sorter.emplace_back(grid[i], i);
+        } else if (min_nodes == 0) {
+            // This will add all tiles which contain no nodes. It is useful
+            // if you want a list of all tiles an a list how "large" each tile is.
+            sorter.emplace_back(0, i);
         }
```

Comment:
`if (grid[i] >= min_nodes) {` ? That would also get rid of the if-condition in new line 185-188.